core/ecschema-editing/src/Editing/Properties.ts

```diff
@@ -38,7 +38,7 @@ export class Properties {
         throw new SchemaEditingError(ECEditingStatus.SetPropertyName, new PropertyId(this.ecClassType, classKey, propertyName), e);
       });
 
-    const baseProperty = await existingProperty.class.getProperty(newPropertyName, true) as MutableProperty;
+    const baseProperty = await existingProperty.class.getProperty(newPropertyName) as MutableProperty;
     if (baseProperty) {
```

Comment:
baseProperty is a very confusing name here.  This is a property with the new name, either from this class or from a base class.

Reply:
Yeah, that was confusing. Fixed.